doc/src/manual/functions.md

```diff
@@ -140,6 +146,31 @@ Int8
 This function will always return an `Int8` regardless of the types of `x` and `y`.
 See [Type Declarations](@ref) for more on return types.
 
+Incidentally, the type of the arguments can also be specified similarly,
+but this is the topic of the [Methods](@ref) chapter.
+
+### Returning nothing
+
+Some functions are used only for their side effects, and do not need to return a value.
+In these cases, the Julia convention is to return the value [`nothing`](@ref):
```

Comment:
This is slightly at odds with the convention to return the modified object as in

```
julia> a = rand(2); push!(a, 2.0)
3-element Array{Float64,1}:
 0.7926734563461015
 0.6590165997556594
 2.0 
```

Reply:
Indeed. In fact I copy pastd this fragment from the [FAQ](https://docs.julialang.org/en/v1/manual/faq/). What do you thinkg of:
```
For functions which do not need to return a value, the Julia convention is to return the value nothing
...
```